docs/meetings/community/2022-11-15.md

```diff
@@ -0,0 +1,113 @@
+---
+tags: binder, meeting, notes
+---
+
+# JupyterHub and BinderHub Team Meeting
+
+- **Date:** Tuesday 15th November 2022
+- **Time:** 6PM UTC
```

Comment:
Just one note: it seems there has been something wonky happening with the meeting time maybe related to DST.

I thought it was at 7PM CET for me yesterday (UTC+2 with DST) but when I checked the hackmd, it was noted as 5PM UTC so 6 for me (following the arewemeetingyet link and I forgot that we reverted to UTC+1). However when I connected, there was another fine gentleman alone there and we were both wondering whether we had missed something.

I then checked the google calendar on [the meeting page on readthedoc](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings/index.html) which showed the meeting being at 10AM.

I wanted to check the October meeting notes to see if I had missed something but I couldn't find them. Now that I have rechecked the google calendar, it seems that is shows the time for North America, hence the confusion with regard to that part.